src/navigations/MainNavigation.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import FeedScreen from '~/screens/FeedScreen';
+import SearchScreen from '~/screens/SearchScreen';
+import HotScreen from '~/screens/HotScreen';
+import MyScreen from '~/screens/MyScreen';
+import { MainTabParamList } from '~/types/navigation';
+
+const Tab = createBottomTabNavigator<MainTabParamList>();
+
+const MainNavigation: React.FC = () => {
+  const getIcon: Function = (focused: boolean, value: string): JSX.Element => (
+    <Ionicons size={25} name={focused ? value : `${value}-outline`} />
+  );
```

Comment:
getIcon 리턴타입 Function 빠져도 될거 같아요
JSX.Element 대신 React.ReactNode 쓰는게 좋아보입니다

```suggestion
  const getIcon = (focused: boolean, value: string): React.ReactNode => (
    <Ionicons size={25} name={focused ? value : `${value}-outline`} />
  );
```